isaric/parsers/ncov-phosp.toml

```diff
@@ -0,0 +1,803 @@
+  phase = "admission"
+  is_present = { field = "crf1a_fatigue", ref = "Y/N" }
+
+[[observation]]
+  name = "fatigue_malaise"
+  date = { field = "patient_sq_date" }
+  phase = "followup"
+  is_present = { field = "psq_fatigue", ref = "Y/N" }
+
+[[observation]]
+  name = "headache"
+  date = { field = "crf1a_date_adm" }
+  phase = "admission"
+  is_present = { field = "crf1a_headache", ref = "Y/N" }
+
+[[observation]]
+  name = "headache"
+  date = { field = "patient_sq_date" }
+  phase = "followup"
+  is_present = { field = "headache", ref = "Y/N" }
+
+[[observation]]
+  name = "history_of_fever"
+  date = { field = "crf1a_date_adm" }
+  phase = "admission"
+  is_present = { field = "crf1a_fever_history", ref = "Y/N" }
+
+[[observation]]
+  name = "history_of_fever"
+  date = { field = "patient_sq_date" }
+  phase = "followup"
+  is_present = { field = "psq_feverish", ref = "Y/N" }
+  
+[[observation]]
+  name = "inability_to_walk_scale"
+  date = { ref = "eq5dDates" } # Q. Check if date is ok?'
+  phase = "followup"
+  value.field = "eq5d5l_q1"
+  [observation.value.values]
+    1 = 1 # I have no problems in walking about
+    2 = 2 # I have slight problems in walking about
+    3 = 2 # I have moderate problems in walking about
+    4 = 3 # I have severe problems in walking about
+    5 = 4 # I am unable to walk about
+
+[[observation]]
+  name = "inability_to_walk"
+  date = { field = "patient_sq_date" }
+  phase = "followup"
+  is_present = { field = "patient_sq_l_t_walking", ref = "Y/N" }
+
+[[observation]]
+  name = "joint_pain"
+  date = { field = "crf1a_date_adm" }
+  phase = "admission"
+  is_present = { field = "crf1a_join_pain", ref = "Y/N" }
+
+[[observation]]
+  name = "joint_pain"
+  date = { field = "patient_sq_date" }
+  phase = "followup"
+  is_present = { field = "joint_pain_or_swelling", ref = "Y/N" }
+
+[[observation]]
+  name = "conjunctivitis"
+  phase = "admission"
+  date = { field = "crf1a_date_adm" }
+  is_present = { field = "crf1a_conjunctivitis", ref = "Y/N" }
+
+[[observation]]
+  name = "loss_of_smell"
+  date = { field = "crf1a_date_adm" }
+  phase = "admission"
+  is_present = { field = "crf1a_loss_smell", ref = "Y/N" }
+
+[[observation]]
+  name = "loss_of_taste"
+  date = { field = "crf1a_date_adm" }
+  phase = "admission"
+  is_present = { field = "crf1a_loss_taste", ref = "Y/N" }
+
+[[observation]]
+  name = "loss_of_taste"
+  date = { field = "patient_sq_date" }
+  phase = "followup"
+  is_present = { field = "loss_of_taste", ref = "Y/N" }
+
+[[observation]]
+  name = "loss_of_smell_or_taste"
+  date = { field = "crf1a_date_adm" }
+  phase = "admission"
+
+  [observation.is_present]
+    combinedType = "any"
+    fields = [
+      { field = "crf1a_loss_taste", ref = "Y/N" },
+      { field = "crf1a_loss_smell", ref = "Y/N" },
+    ]
+
+[[observation]]
+  name = "loss_of_smell_or_taste"
+  date = { field = "patient_sq_date" }
+  phase = "followup"
+  is_present = { field = "loss_of_smell_or_taste" }
```

Comment:
@ekamau should this combine loss_of_sense_of_smell and loss_of_taste from the PSQ? I can't see a variable named loss_of_smell_or_taste

Reply:
yes - this should combine followup PSQ fields for smell and taste ... there was an error - there was no field name 'loss_of_smell_or_taste' in the data.